docs/sources/mimir/operators-guide/configure/configure-otel-collector.md

```diff
@@ -102,3 +102,23 @@ service:
       processors: [...]
       exporters: [..., otlphttp]
 ```
+
+## Format considerations
+
+We currently follow the official specification described here: [OTLP Metric points to Prometheus](https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#otlp-metric-points-to-prometheus).
+
+You might experience the following common issues:
+
+#### Dots (.) are converted to \_
+
+Because Prometheus metrics do not support `.` and `-` characters in metric or label names, they get converted to `_` in Prometheus.
+
+For example:
+
+`requests.duration{http.status_code=500, cloud.region=us-central1}` in OTLP becomes `requests_duration{http_status_code=”500”, cloud_region=”us-central1”}` in Prometheus.
+
+#### Resource attributes are added to the `target_info` metric.
+
+For details, see the [OpenTelemetry Spec](https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#resource-attributes).
+
+However, `<service.namespace>/<service.name>` or `<service.name>` (if the namespace is empty), is added as the label `job`, and `service.instance.id` is added as the label `instance` to every metric.
```

Comment:
```suggestion
* Dots (.) are converted to \_
   Prometheus metrics do not support `.` and `-` characters in metric or label names. Prometheus converts these characters to `_`. For example:

   `requests.duration{http.status_code=500, cloud.region=us-central1}` in OTLP becomes 
   `requests_duration{http_status_code=”500”, cloud_region=”us-central1”}` in Prometheus.

* Resource attributes are added to the `target_info` metric.

   For details, see the [OpenTelemetry Resource Attributes](https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#resource-attributes) specification.

  However, `<service.namespace>/<service.name>` or `<service.name>` (if the namespace is empty), is added as the label `job`, and `service.instance.id` is added as the label `instance` to every metric.
```
Is `instance` indeed intended to be verbatim?

Reply:
I have not tested this locally, so make sure that the context about each point hangs under the respective list item.

Reply:
> Is instance indeed intended to be verbatim?

Yes